pkg/commands/configure/configure_test.go

```diff
@@ -115,6 +118,9 @@ func TestConfigure(t *testing.T) {
 
 [language]
 
+  [language.javascript]
```

Comment:
A bit confused about how this would work, in reference to the above comment about multiple projects. Surely this should be configured on a per-project level? (or inferred from the lockfiles)